experimenter/experimenter/experiments/migrations/0243_no_feature_config_monitor.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 3.1.8 on 2021-04-29 17:22
```

Comment:
We do not need a tombstone feature for web applications.

We only need tombstone features for applications that did not always support multi-feature experiments. Since Cirrus will starts with supporting multi-feature experiments, we have no reason to serialize a payload with a tombstone feature.